src/Form/FormAnalyzer.js

```diff
@@ -46,9 +46,9 @@ class FormAnalyzer {
         shouldCheckUnifiedForm = false, // Should check for login/signup forms
         shouldBeConservative = false // Should use the conservative signup regex
     }) {
-        const negativeRegex = new RegExp(/sign(ing)?.?in(?!g)|log.?in/i)
+        const negativeRegex = new RegExp(/sign(ing)?.?in(?!g)|log.?in|unsubscri/i)
```

Comment:
Add support for unsubscribe forms. See https://app.asana.com/0/1200930669568058/1201641487236453/f.